### DISCUSSION TITLE
Звуки уведомлений

### DISCUSSION THREAD

**Discussion by mbs0ft** [OpenVK/openvk#55] [Ideas] [answered]
Если использовать официальные звуки, то вот оригиналы этих уведомлений:
Входящее сообщение: https://st.fl.ru/users/bbfunk/upload/f_4dc7efd744e39.mp3
Уведомление: https://st.fl.ru/users/bbfunk/upload/f_4dc7efc7b9ca3.mp3

Так же Дуров отмечал другие звуки, которые были в рассмотрении:
https://st.fl.ru/users/ya_kareshka/upload/f_4dc844ce14c7d.mp3
https://st.fl.ru/users/ya_kareshka/upload/f_4dbdb8d714bff.mp3

_proofpic_
![image](https://user-images.githubusercontent.com/22106304/104644533-8af0b700-56b6-11eb-88bc-dbcdd7696b68.png)

**Comment by WerySkok** [comment]
Конечно, у OpenVK и так большие проблемы с лицензионной чистотой, но думаю, что эти звуки тут явно не подойдут.

**Reply by mbs0ft** [reply]
поэтому я и добавил другие ссылки

**Comment by ghost** [comment, marked as answer]
Надо бы сделать... когда найду время для этого, то, может быть, что-нибудь придумаю.

UPD: Сделано.

**Reply by ghost** [reply]
Уже придумал в d16b5fdf339ebead36165392086586255e5d5b64
